charts/argo-workflows/templates/controller/workflow-role.yaml

```
@@ -18,6 +18,9 @@ rules:
     verbs:
       - create
       - patch
+    {{- with $.Values.workflow.rbac.rules }}
+      {{- toYaml . | nindent 2 }}
+    {{- end }}
```

nits; indent is wrong?
```suggestion
  {{- with $.Values.workflow.rbac.rules }}
    {{- toYaml . | nindent 2 }}
  {{- end }}
```

---

nice catch, thanks.